### PR TITLE
ping, ping6: add German translation

### PR DESCRIPTION
**Pull request by LukBukkit**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** Not known

### CODE REVIEW

File: pages.de/common/ping.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+- Sende Pings an ein Gerät im Netzwerk:
+
+`ping {{ziel}}`
+
+- Sende nur eine bestimmte Anzahl an Pings:
+
+`ping -c {{anzahl}} {{ziel}}`
+
+- Sende Pings und bestimme das Interval in Sekunden zwischen diesen, standardmäßig ist es eine Sekunde:
```
Comment (pixelcmtd) [inline pages.de/common/ping.md:14]:
```suggestion
- Sende Pings und bestimme das Interval in Sekunden zwischen diesen (standardmäßig ist es eine Sekunde):
```

File: pages.de/common/ping.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`ping -i {{sekunden}} {{ziel}}`
+
+- Sende Pings ohne symbolische Namen für Adressen aufzulösen:
+
+`ping -n {{ziel}}`
+
+- Sende Pings und signalisiere eine erfolgreiche Antwort durch ein Signal in der Konsole, diese muss das Signal unterstützen:
```
Comment (pixelcmtd) [inline pages.de/common/ping.md:22]:
```suggestion
- Sende Pings und signalisiere eine erfolgreiche Antwort durch ein Bell Signal (wenn das Terminal es unterstützt):
```

File: pages.de/common/ping6.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Sende Pings an ein Gerät im Netzwerk:
+
+`ping6 {{ziel}}`
+
+- Sende nur eine bestimmte Anzahl an Pings:
+
+`ping6 -c {{anzahl}} {{ziel}}`
+
+- Sende Pings und bestimme das Interval in Sekunden zwischen diesen, standardmäßig ist es eine Sekunde:
```
Comment (pixelcmtd) [inline pages.de/common/ping6.md:14]:
```suggestion
- Sende Pings und bestimme das Interval in Sekunden zwischen diesen (standardmäßig ist es eine Sekunde):
```

File: pages.de/common/ping6.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+`ping6 -i {{sekunden}} {{ziel}}`
+
+- Sende Pings ohne symbolische Namen für Adressen aufzulösen:
+
+`ping6 -n {{ziel}}`
+
+- Sende Pings und signalisiere eine erfolgreiche Antwort durch ein Signal in der Konsole, diese muss das Signal unterstützen:
```
Comment (pixelcmtd) [inline pages.de/common/ping6.md:22]:
```suggestion
- Sende Pings und signalisiere eine erfolgreiche Antwort durch ein Bell Signal (wenn das Terminal es unterstützt):
```

File: pages.de/common/ping.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+- Sende nur eine bestimmte Anzahl an Pings:
+
+`ping -c {{anzahl}} {{ziel}}`
+
+- Sende Pings und bestimme das Interval in Sekunden zwischen diesen, standardmäßig ist es eine Sekunde:
+
+`ping -i {{sekunden}} {{ziel}}`
+
+- Sende Pings ohne symbolische Namen für Adressen aufzulösen:
```
Comment (pixelcmtd) [inline pages.de/common/ping.md:18]:
```suggestion
- Sende Pings ohne symbolische Namen nach Adressen aufzulösen:
```

File: pages.de/common/ping6.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+- Sende nur eine bestimmte Anzahl an Pings:
+
+`ping6 -c {{anzahl}} {{ziel}}`
+
+- Sende Pings und bestimme das Interval in Sekunden zwischen diesen, standardmäßig ist es eine Sekunde:
+
+`ping6 -i {{sekunden}} {{ziel}}`
+
+- Sende Pings ohne symbolische Namen für Adressen aufzulösen:
```
Comment (pixelcmtd) [inline pages.de/common/ping6.md:18]:
```suggestion
- Sende Pings ohne symbolische Namen nach Adressen aufzulösen:
```

**Comment by LukBukkit** [comment]
Thank you for your suggestions, I've applied them to my changes :)

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**